.gitignore

```diff
@@ -0,0 +1 @@
+**/node_modules
```

Comment:
[Optional] Please consider removing the `node_modules` directory from your online repository since this makes your repo look unprofessional you can do this by following the following steps in `vscode`.
- `$ git rm -r --cached node_modules` 
- Then add node_modules to the `.gitignore`
- Commit the changes and push online